### DISCUSSION TITLE
Web GUI Not Opening - Browser Times Out

### DISCUSSION THREAD

**Discussion by Mohammedalkahala** [Security-Onion-Solutions/securityonion#13014] [2.4]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8 cores

### RAM

16gb

### Storage for /

200gb

### Storage for /nsm

125gb

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I am currently experiencing an issue with Security Onion where the web GUI does not load, and the browser times out when attempting to log in. Here are my trouble shooting steps:
Verified Security Onion installation and configuration.
Checked the status of Docker containers.
Reviewed NGINX configuration and logs.
Ensured correct IPtables rules and network configuration.
Attempted to access the GUI using different browsers and cleared cache.
Confirmed DNS resolution and connectivity to the Security Onion server.
Rebooted the server and confirmed services are running correctly.

Please see all of my screen shots to understand the error messages I get, and how my system seems to be running. 
![SOwebbrowserpage](https://github.com/Security-Onion-Solutions/securityonion/assets/109744546/d6473d3f-9c12-482f-b136-0820794415a7)
![SOhardware](https://github.com/Security-Onion-Solutions/securityonion/assets/109744546/472fd122-703d-487d-8c72-54036a52e2fd)
![SONslookup](https://github.com/Security-Onion-Solutions/securityonion/assets/109744546/e5167d0b-f011-4875-9ce2-14a2da007c05)
![SOcurl](https://github.com/Security-Onion-Solutions/securityonion/assets/109744546/ee95a638-c3c8-452c-a69f-d6f722946663)
![soifconfigP2](https://github.com/Security-Onion-Solutions/securityonion/assets/109744546/796289b5-8bfd-4140-a2fa-1dcd431694c1)
![soifconfigP1](https://github.com/Security-Onion-Solutions/securityonion/assets/109744546/33dd1e12-4851-43b6-8fe0-308a455d8561)
![SOerrormessage](https://github.com/Security-Onion-Solutions/securityonion/assets/109744546/92b6fbaf-6b18-4770-9ba9-ae728d1a5b7d)
![SOtest](https://github.com/Security-Onion-Solutions/securityonion/assets/109744546/a81c11b1-d36f-4af4-b651-93905d42c776)
![SOversion](https://github.com/Security-Onion-Solutions/securityonion/assets/109744546/9b1d3859-5aa8-465d-9ba7-e16fefdb1824)
![SOsostatus](https://github.com/Security-Onion-Solutions/securityonion/assets/109744546/8d7dd838-5430-4143-8cf0-0fa3882c504e)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
Check to make sure that the IP address of your browser machine is allowed to connect through the host-based firewall:
https://docs.securityonion.net/en/2.4/post-installation.html#adjust-firewall-rules

For more information, please see:
https://docs.securityonion.net/en/2.4/firewall.html#host-firewall

**Reply by Mohammedalkahala** [reply]
It worked thanks so much.

**Comment by Mohammedalkahala** [comment]
Check to make sure that the IP address of your browser machine is allowed to connect through the host-based firewall:
https://docs.securityonion.net/en/2.4/post-installation.html#adjust-firewall-rules

For more information, please see:
https://docs.securityonion.net/en/2.4/firewall.html#host-firewall
